tests/src/test_projects/signed_i128/tests/mod.rs

```diff
@@ -16,6 +16,8 @@ mod success {
 
         let instance = Testi128::new(wallet, path_to_bin);
 
-        let _result = instance.main().call().await;
+        let params = TxParameters::new(Some(1), Some(10000000), None);
+        let _result = instance.main().tx_params(params).call().await;
+        assert_eq!(_result.is_err(), false);

```

Comment:
The underscore is used for unused variables.
Same suggestion for other instances of this usage.

```suggestion
        let result = instance.main().tx_params(params).call().await.unwrap();
        assert!(result.value);

```